README.md

```diff
@@ -71,7 +88,7 @@ For full documentation, check out the README.md for each package:
 - [Torus](packages/torus/README.md)
 - [WalletConnect](packages/walletconnect/README.md)
 - [WalletLink](packages/walletlink/README.md)
```

Comment:
Do we want to swap WalletLink out for Coinbase?

Reply:
Yeah good call.